examples/docker-asyncio/Pipfile

```diff
@@ -0,0 +1,16 @@
+[[source]]
```

Comment:
I know this is just an example but based on my experiences, I would recommend against using pipenv for anything. It's extremely slow and almost impossible to do clean upgrades (i.e. just updating one sub-dependency). I'd stick with standard python requirements files -- and virtualenvs should not be required inside a docker environment.

Reply:
Unfortunately, I've been using pipenv files for most of the examples. It has actually been surprisingly easy for me to pick my work back up. Maybe a separate issue of de-pipenv-ifying this repository would be in order.